Allure.NUnit/Core/AllureNUnitAttribute.cs

```diff
@@ -18,15 +18,21 @@ public AllureNUnitAttribute(bool wrapIntoStep = true)
 
         public void BeforeTest(ITest test)
         {
-            _allureNUnitHelper.Value = new AllureNUnitHelper(test);
-            _allureNUnitHelper.Value.StartAll(_isWrappedIntoStep);
+            var value = new AllureNUnitHelper(test);
+
+            _allureNUnitHelper.AddOrUpdate(test.FullName, value, (key, existing) => value);
```

Comment:
How about `test.Id` instead of `test.FullName`? It's shorter but still unique.

Reply:
Done.